.github/workflows/deploy-dojo-for-e2e.yml

```
@@ -0,0 +1,316 @@
+        with:
+          version: "9.5"
+
+      - name: (Dojo) Use Node.js 22
+        if: matrix.app.type == 'dojo'
+        uses: actions/setup-node@v2
+        with:
+          node-version: 22.x
+
+      - name: (Dojo) Install CPK Monorepo
+        if: matrix.app.type == 'dojo'
+        run: |
```

actionlint (1.7.7)

string should not be empty

(syntax-check)